.github/workflows/ci.yml

```
@@ -0,0 +1,70 @@
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Make application-secret.properties
+        run: |
+          mkdir -p src/main/resources
+          echo "$APPLICATION" > src/main/resources/application-secret.properties
+        env:
+          APPLICATION: ${{ secrets.APPLICATION }}
+
+      - name: Create GCP credentials file
+        if: ${{ secrets.GCP_CREDENTIALS_BASE64 != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)